examples/discordjs-typescript/src/index.ts

```diff
@@ -0,0 +1,16 @@
+import { Client, Message } from 'discord.js';
+const client = new Client();
+
+client.on('ready', () => {
+  console.log(`Logged in as ${client.user?.tag}!`);
+});
+
+client.on('message', (msg: Message) => {
```

Comment:
```suggestion
client.on('message', (msg) => {
```
Is this type annotation needed?

Reply:
That is what typescript is for

Reply:
TypeScript auto detects it.

Reply:
that still doesn’t mean we dont have to give the annotation